fabric/0001-Integrate-tlcc-syscc.patch

```diff
@@ -0,0 +1,47 @@
+From 0b29c14088c10e64ecaf918c9a96e5444d106e47 Mon Sep 17 00:00:00 2001
```

Comment:
Thinking a small amount forward, modules plus a patch against upstream is probably difficult.

It might be possible with a vendor directory, but requires more investigation in the future.

Reply:
Good point! I think the patch is not a long term version, we may try to find / create better integrations points in the future.

Once modules in Fabric are upstream we may use `replace` to point to our local patched version?!

Reply:
hmm, not sure how modules or vendor will help regarding the patch itself?  It only would help making the building of the patch (potentially) easier/cleaner as the symlinking we currently do?

Reply:
@MHBauer What i meant is i doubt we get away with _any_ patch.  This patch is as minimal as it gets and nothing with modules or vendor aleviates the need for the patch itself (only could simplify make cleaner tying in and build the related FPC code) ...